### PR TITLE
setup releasing to PyPI

### LINKED ISSUE #63: provide conu via PyPI

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #63

TODO:
* [x] remove \_\_version\_\_ from `apidefs/backend`

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/tpas](https://da.gd/tpas)__
* Comment: __See details for logs__

File: setup.py [hunk 1]
```diff
@@ -36,10 +36,15 @@ def get_requirements():
 
 data_files = {}
 
+# https://packaging.python.org/guides/single-sourcing-package-version/
+version = {}
+with open("./conu/version.py") as fp:
```
Comment (dhodovsk) [inline setup.py]: Please, add this also to docs/source/conf.py file (https://github.com/fedora-modularity/conu/blob/master/docs/source/conf.py#L59)

**Comment by dhodovsk** [comment]
@bkabrda could you please review? You are more experienced in this topic.

File: setup.py (hunk 1 above)
Comment (TomasTomecek) [reply setup.py]: I sure can!

**Comment by TomasTomecek** [comment]
updated

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/cBWht](https://da.gd/cBWht)__
* Comment: __See details for logs__

**Comment by bkabrda** [comment]
FWIW this PR LGTM ;) 👍

**Comment by TomasTomecek** [comment]
rebased and hoping for good test results

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/mN1X](https://da.gd/mN1X)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/msoi](https://da.gd/msoi)__
* Comment: __See details for logs__

**Pull request merged**